### PR TITLE
Responsive fixes

### PR DESCRIPTION
**Pull request by maxharrisnet**
We worked as a group to identify bugs and fix them. These include Accessibility and Responsive adjustments. 
- Land Acknowledgment modal
- Footer
- Image gallery
- Banner
- Form layout

### CODE REVIEW

**Review by SolivanLau — APPROVED** [review]
Looks great!

File: css/style.css [hunk 1]
```diff
@@ -1,743 +1,760 @@
...
+	height: 2px;
+	background-color: var(--hr-color);
 }
 /* global wrapper class */
 .wrapper {
-  width: var(--wrapper-width);
-  max-width: 1440px;
-  margin: 0 auto;
+	width: var(--wrapper-width);
+	max-width: 1230px;
```
Comment (maxharrisnet) [inline css/style.css:139]: Changed this to match the Figma design, discovered it was causing issues with the form layout.

Comment (SolivanLau) [reply css/style.css:139]: didn't notice the width of the figma file - great point!

**Comment by maxharrisnet** [comment]
Add a comment to the bottom of the conversation section!

File: assets/images/itlct-gallery-9.png
Comment (lilcheerio) [inline assets/images/itlct-gallery-9.png:1]: testing comments. Is this okay?

File: css/style.css [hunk 2]
```diff
@@ -1,743 +1,760 @@
 /* BOX SIZING */
-html {
```
Comment (maxharrisnet) [inline css/style.css:2]: Just moving this down a few lines after the variables

**Comment by liannarice** [comment]
does the text on the image next to the form look a bit different? in the figma it looks like the text doesn't cover the bottom part of the image (because its lighter in colour and #accessibility ). Is it because we adjusted the font weight?

**Comment by roguemacaroon** [comment]
Super happy with the fixes. would have loved to see us get 100 on the performance, but that can be a future goal

File: css/style.css (hunk 2 above)
Comment (CosmicStarfire) [reply css/style.css:2]: Awesome pawsome!

**Comment by lilcheerio** [comment]
Everything looks really good!

File: css/style.css [hunk 3]
```diff
@@ -1,743 +1,760 @@
...
-  background-image: url("../assets/images/contact-form.png");
-  background-position: center;
-  background-size: cover;
+	width: 432px;
+	min-height: 597px;
+	border-radius: 4px;
+	background: var(--bg-color);
+	padding: 20px;
+	background-image: url('../assets/images/contact-form.png');
+	background-color: var(--dark-teal);
```
Comment (maxharrisnet) [inline css/style.css:552]: Added background to fix color contrast issue with WAVE. Does this look right?

File: assets/images/itlct-gallery-9.png
Comment (maxharrisnet) [reply assets/images/itlct-gallery-9.png:1]: Perfect - thanks for trying it! 🐶

File: index.html [hunk 4]
```diff
@@ -1,527 +1,511 @@
...
+    </button>
+  </div>
+  <!-- header -->
+  <header>
+    <!-- wrapper -->
+    <div class="wrapper">
+      <!-- navigation -->
+      <nav class="header-nav">
+        <div class="logohead">
+          <a href="/" aria-label="Return to homepage">
```
Comment (SolivanLau) [inline index.html:41]: Great use of aria-label - i am sure this is what contributed to the 100% accessibility score

File: css/style.css [hunk 5]
```diff
@@ -748,139 +765,252 @@ footer nav {
...
+	}
+
+	.modal-scroll {
+		max-height: calc(100vh - 260px);
+	}
+
+	/* footer styles */
+
+	/* Land Acknowledge button placement */
+	.btn-la {
```
Comment (charlisable) [inline css/style.css:1013]: button could be positioned a little bit to be centered

**Review by roo-tech — COMMENTED** [review]
The correction to the modal.js for the extra EventListener for the extra "keydown" was a good catch.

**Comment by maxharrisnet** [comment]
> does the text on the image next to the form look a bit different? in the figma it looks like the text doesn't cover the bottom part of the image (because its lighter in colour and #accessibility ). Is it because we adjusted the font weight?

@liannarice I think you are right, good call! We just made some adjustments in class together. I just pushed the changes up. Please take another look :)

File: css/style.css [hunk 6]
```diff
@@ -748,139 +765,260 @@ footer nav {
...
-@media screen and (max-width: 850px) {
-  /* contact form section */
-  /* adjusting first and last name inputs */
-  .name-input-container {
-    flex-direction: column;
-  }
-  .name-inputs {
-    width: 100%;
-  }
+@media screen and (max-width: 1100px) {
```
Comment (maxharrisnet) [inline css/style.css:768]: Hey, just wondering why this changed?

File: css/style.css [hunk 7]
```diff
@@ -748,139 +765,260 @@ footer nav {
...
+	}
+
+	.header {
+		background-image: url('../assets/mobile/hero-image.png');
+	}
+
+	/* who we are section */
+
+	.ecosystem-banner {
+		background-image: url('../assets/mobile/global-ecosystem.png');
```
Comment (maxharrisnet) [inline css/style.css:805]: Can you add a comment to the CSS to clarify the purpose of this code?

**Comment by maxharrisnet** [comment]
> Loving the new changes, The footer is looking pretty great. I wish we had time to change the scrolling photos to actual scrolling photos, like manually. Because just having them stagnant, to me feels incomplete. I understand we're waiting to change this, so I am very excited for this change to happen when it does.

Thanks for your hard work and feedback on the gallery stuff @CosmicStarfire !

File: css/style.css (hunk 5 above)
Comment (maxharrisnet) [reply css/style.css:1021]: We just updated it, I think it's looking better

Screen Shot 2023-12-11 at 12 27 25 PM

**Review by SolivanLau — APPROVED** [review]
looks good!

**Review by CosmicStarfire — APPROVED** [review]
Awesome Pawsome Sauce Max! Looking pretty perf

**Pull request merged**